apps/emqx_connector/src/emqx_connector_mqtt.erl

```diff
@@ -112,10 +120,9 @@ on_stop(InstId, #{channels := NameList}) ->
 on_query(_InstId, {create_channel, Conf}, _AfterQuery, #{name_prefix := Prefix,
         baisc_conf := BasicConf}) ->
     create_channel(Conf, Prefix, BasicConf);
-on_query(InstId, {publish_to_local, Msg}, _AfterQuery, _State) ->
-    logger:debug("publish to local node, connector: ~p, msg: ~p", [InstId, Msg]);
-on_query(InstId, {publish_to_remote, Msg}, _AfterQuery, _State) ->
-    logger:debug("publish to remote node, connector: ~p, msg: ~p", [InstId, Msg]).
+on_query(InstId, {send_msg, ChannelName, Msg}, _AfterQuery, _State) ->
+    logger:debug("send msg on channel: ~p, msg: ~p", [ChannelName, Msg]),
+    .
```

Comment:
`ok.` ?

Reply:
This is a tmp push to save the daily work. It has been updated.